lms/djangoapps/teams/static/teams/templates/edit-team.underscore

```diff
@@ -38,7 +70,7 @@
     <button class="action action-primary">
         <%=
             interpolate_text(
-                _.escape(gettext("{primaryButtonTitle} {span_start}a new team{span_end}")),
```

Comment:
Here I see the translation of the primaryButtonTitle. This won't work because you can't call gettext on a variable, since that prevents the actual text represented by the variable from being scraped.